azurerm/provider.go

```diff
@@ -88,13 +88,15 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_dns_txt_record":   resourceArmDnsTxtRecord(),
 			"azurerm_dns_zone":         resourceArmDnsZone(),
 
+			"azurerm_eventgrid_topic":             resourceArmEventGridTopic(),
 			"azurerm_eventhub":                    resourceArmEventHub(),
 			"azurerm_eventhub_authorization_rule": resourceArmEventHubAuthorizationRule(),
 			"azurerm_eventhub_consumer_group":     resourceArmEventHubConsumerGroup(),
 			"azurerm_eventhub_namespace":          resourceArmEventHubNamespace(),
 			"azurerm_express_route_circuit":       resourceArmExpressRouteCircuit(),
-			"azurerm_image":                       resourceArmImage(),
-			"azurerm_key_vault":                   resourceArmKeyVault(),
+
+			"azurerm_image":     resourceArmImage(),
+			"azurerm_key_vault": resourceArmKeyVault(),
```

Comment:
Given a lot of the resources share the same initial character, the resources are _vaguely_ grouped by letter so it was easier to skim-read, but happy to move them up?